index.ts

```diff
@@ -365,8 +366,10 @@ export interface JUserService {
   getLocale(): string
   getPreference(name: string): string | null
   hasPreference(name: string): boolean
+  getDistanceUnit(projectId: number): JDistanceUnit | null
```

Comment:
The function should be : getDistanceUnit(): JDistanceUnit
But if no project is selected you throw an error that say "No project selected, cannot get the distanceUnit".
Also don't return null because by default there is a distanceUnit.